wgpu/backends/wgpu_native/_api.py

```diff
@@ -1079,6 +1067,8 @@ class GPUDevice(classes.GPUDevice, GPUObjectBase):
     # GPUObjectBaseMixin
     _release_function = libf.wgpuDeviceRelease
 
+    CREATE_PIPELINE_ASYNC_NOT_IMPLEMENTED = True
```

Comment:
Let's make this private (prefix with an underscore).

Reply:
Done